lib/logzio-nodejs.js

```diff
@@ -24,6 +24,7 @@ var LogzioLogger = function (options) {
     this.timer = null;
     this.closed = false;
     this.supressErrors = options.supressErrors || false;
+    this.addNanoSecs = options.addNanoSecs || false;
```

Comment:
I think we should document this option in the code and/or the README where we define all the parameters